### PR TITLE
SEAR-267 Application SHOULD NOT Require Exact Match of Facility Name in order to Show in Results

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/SEAR-267

#### Brief feature description
- partial search results using the facility name and facility search

#### Does this PR block/effect other stories?
-n/a

#### Do any commands need to be run in order to run the project with this PR?
-n/a

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

**Comment by Nasir2712** [comment]
jenkins, retest this please

File: app/classes/elastic/query_builder.rb [hunk 1]
```diff
@@ -3,15 +3,22 @@ class Elastic::QueryBuilder
...
     # each key is an attribute to query on
     query_hash.each do |k, v|
-      query_leaves << { v[:query_type].to_sym => { k.to_sym => v[:value] } }
+      query_leaves << name_and_other_queries(k, v)
     end
 
     return query_leaves
   end
 
+  def self.name_and_other_queries(k, v)
```
Comment (ratneshraval) [inline app/classes/elastic/query_builder.rb]: I think better name is `query_by_type` or something like that.

File: app/javascript/search/common/commonUtils.js [hunk 2]
```diff
@@ -27,6 +27,17 @@ export const checkForValue = (value) => {
   return value || undefined
 }
 
+export const checkAndSplitValue = (value) => {
+  if (value !== undefined || '') {
+    return value.split(' ').map((val) => {
```
Comment (ratneshraval) [inline app/javascript/search/common/commonUtils.js]: Let's use `filter` instead of `map + regex`

Comment (Nasir2712) [reply app/javascript/search/common/commonUtils.js]: done

File: app/classes/elastic/query_builder.rb (hunk 1 above)
Comment (Nasir2712) [reply app/classes/elastic/query_builder.rb]: done

File: app/javascript/search/common/commonUtils.js [hunk 3]
```diff
@@ -27,6 +27,13 @@ export const checkForValue = (value) => {
   return value || undefined
 }
 
+export const checkAndSplitValue = (value) => {
+  if (value !== undefined || '') {
+    const filteredSet = ['*', '']
```
Comment (ratneshraval) [inline app/javascript/search/common/commonUtils.js]: 👍

**Pull request merged**